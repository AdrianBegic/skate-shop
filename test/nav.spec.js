import { mount } from '@vue/test-utils';
import NavBar from '../components/nav.vue'


describe('NavBar component', () => {
    describe('Default mounting properties', () => {
        it('should mount successfully', () => {
            const wrapper = mount(NavBar);

            expect(wrapper.vm._isMounted).toBeTruthy();

            expect(wrapper).toMatchSnapshot();
        });

        it('should unmount successfully', () => {
            const wrapper = mount(NavBar);

            const NavBarInstance = wrapper.vm;
            NavBarInstance.$destroy();
            expect(NavBarInstance._isDestroyed).toBeTruthy();

            expect(wrapper).toMatchSnapshot();
        });

        it('should mount links with proper paths', () => {
            const
        })
    })
})