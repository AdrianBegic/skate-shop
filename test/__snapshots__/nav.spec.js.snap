// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component Default mounting properties should mount successfully 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;

exports[`NavBar component Default mounting properties should unmount successfully 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
